[
  {
    "poster": {
      "poster_name": "B Bahrami",
      "poster_handle": "bahramibh7"
    },
    "time_stamp": "2021-08-30 02:08:29",
    "text": "Hi,\nim integration with HubSpot ticket and contact\ni have read the documentation\ni think there is tooooo many api call for create sample ticket, or create reply for ticket , or get new admin's reply\ni also ask question in <https://community.HubSpot.com/t5/CRM/Ticket-integration/m-p/487149#M128743>",
    "reply_count": 1,
    "reply_users_count": 1,
    "replies": [
      {
        "time_stamp": "2021-08-31 04:57:38"
      }
    ]
  },
  {
    "poster": {
      "poster_name": "Ido Lishner",
      "poster_handle": "idol"
    },
    "time_stamp": "2021-08-30 03:41:09",
    "text": "Hello, try to pass an email parameter in webhook, tried to add this to the URL /?email={{contact.email}} but HS says invalid, any ideas?"
  },
  {
    "poster": {
      "poster_name": "Heber Brandao",
      "poster_handle": "heber.brandao"
    },
    "time_stamp": "2021-08-30 04:17:20",
    "text": "Hey everyone, I am running into a a basic issue when implementing my HubSpot-api-python into production. I am trying to batch read a list of companies, but somehow the same snippet works on the HubSpot website (<https://developers.HubSpot.com/docs/api/crm/companies>) but not on my jupyter notebook.\n```import HubSpot\nfrom HubSpot.crm.companies import BatchReadInputSimplePublicObjectId, ApiException\n\nclient = HubSpot.Client.create(api_key=\"REDACTED\")\n\nbatch_read_input_simple_public_object_id = BatchReadInputSimplePublicObjectId(properties=\"HubSpot_owner_id\", id_property=\"item_id\", inputs=[{\"id\":\"7762834\"},{\"id\":\"3245075\"}])\n\ntry:\n    api_response = client.crm.companies.batch_api.read(archived=False, batch_read_input_simple_public_object_id=batch_read_input_simple_public_object_id)\n    pprint(api_response)\nexcept ApiException as e:\n    print(\"Exception when calling batch_api-&gt;read: %s\\n\" % e)```\nWhen I execute it, I get the following response:\n```\"status\":\"error\",\"message\":\"Invalid input JSON on line 1, column 16: Cannot deserialize instance of `java.util.HashSet` out of VALUE_STRING token\",\"correlationId\":\"36920e0a-7d11-4c1a-989e-c95434935d4d\"```\nThe very same request was copy pasted from HubSpot website, but now I cannot find a reason why this doesnt work"
  },
  {
    "poster": {
      "poster_name": "John Fuller - Blames Bugs",
      "poster_handle": "john.ascendinbound"
    },
    "time_stamp": "2021-08-30 11:07:55",
    "text": "There any timeline when v2 will be sunset? Have clients already asking after <https://developers.HubSpot.com/changelog/crm-api-v3-is-generally-available|this news in changelog>",
    "reply_count": 1,
    "reply_users_count": 1,
    "replies": [
      {
        "time_stamp": "2021-08-30 12:49:41"
      }
    ]
  },
  {
    "poster": {
      "poster_name": "Vishal Kumar",
      "poster_handle": "vishalkumar"
    },
    "time_stamp": "2021-08-30 12:49:41",
    "text": "Hey <@U3T21PDBK>,\n\nThe solutions could get upgraded with V3 APIs. A few HS Contact-related APIs are currently under development, which could be a reason for no upgrade right now. On the other hand, Bulk APIs to process data in Bulk could be one of the biggest benefits of migrating on V3."
  },
  {
    "poster": {
      "poster_name": "Sam McKinney",
      "poster_handle": "sam"
    },
    "time_stamp": "2021-08-30 13:29:08",
    "text": "Hi All! Does anyone know if the two rich text editor fields in the Quotes interface will come through as rich text when looking at that specific quote through the API?\n\nI went and tested it on a single quote that I have in my test portal with some text in the Rich Text ‘Comments to Buyer’ field, but I am not seeing that get saved to the actual quote object.\n\nDoes that live somewhere else?"
  }
]